### ISSUE TITLE
lodash dependency

### ISSUE THREAD

**Issue by scott-w** [JSteunou/marionette.morphdom#1]
Was just running through the source and saw a dependency on `lodash` in the `package.json`. Is this a hard dependency for this to work or would `underscore` work just as well?

**Comment by JSteunou** [comment]
Hard dependency. I am using lodash throttle. `some` and `isNumber` are underscore compatible. 
Maybe I should emded those fonctions ? What do you think ?

**Comment by scott-w** [comment]
Underscore has `throttle` too (http://underscorejs.org/#throttle) - is there a difference in their API?

In terms of embedding, I'm assuming most users would have lodash or underscore installed already?

**Comment by JSteunou** [comment]
Lodash throttle can be canceled

**Comment by scott-w** [comment]
Fair enough, that makes sense. I can update the README with a note on that if you like?

**Comment by JSteunou** [comment]
Sure, could be useful !
